server/sonar-server/src/main/java/org/sonar/server/settings/ws/SetAction.java

```diff
@@ -138,6 +216,17 @@ private void validate(SetRequest request, Optional<ComponentDto> component) {
     checkComponentQualifier(request, definition, component);
   }
 
+  private void checkFieldType(SetRequest request, @Nullable PropertyDefinition definition, ListMultimap<String, String> valuesByFieldKeys) {
+    for (PropertyFieldDefinition fieldDefinition : definition.fields()) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) NullPointerException might be thrown as 'definition' is nullable here [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AS2259)